src/middleware.ts

```diff
@@ -5,7 +5,7 @@ import { NextRequest, NextResponse } from 'next/server';
 
 /**
  * Next.js middleware is the code which is run before a request is completed
- * @param request: NextRequest
+ * @param NextRequest
```

Comment:
This notation would be better to match others.
```ts
@param request - NextRequest
```

Reply:
`@params` used throughout the codebase aren't using the dash `-`.
Looks like `@properties` are the ones using the dash.

Do you want to change all params to use the dash too?

Reply:
@jaas666 
Thank you for bringing that to my attention! I appreciate it.

I looked it up in the [official documentation](https://tsdoc.org/pages/tags/param/) and noticed that it uses dashes. Thus, we kindly request updating everything using `-`.

Reply:
Done 👍

Reply:
It seems that there are multiple parts that still remain without a dash. Could you please search for `@param` and fix all of them? Thank you so much!

Reply:
I'm only seeing without a dash the ones that don't have a description. Do you want to add the dash even though there's no description?
For example, do you want to change `@param event` to `@param event -`

Reply:
I am sorry, I was misunderstood. You are totally right!

Reply:
I found one I missed.. committing it now

Reply:
Done